reduce.js

```diff
@@ -0,0 +1,131 @@
+let arrayOne = [1, 2, 3, 4];
+let arrayTwo = [4, 6, 8];
+const even = element => element % 2 === 0;
+
+const concat = (array, newElement) => {
```

Comment:
newElement can be not just an array, please handle cases, when newElement is not array